services/transaction_body.proto

```diff
@@ -317,8 +318,13 @@ message TransactionBody {
     ScheduleSignTransactionBody scheduleSign = 44;
 
     /**
-     * Adds an approved allowance for spender to transfer owner's hbar.
+     * Adds one or more approved allowances for spenders to transfer the paying account's hbar or tokens.
      */
-    CryptoApproveAllowanceTransactionBody cryptoApproveAllowance = 45;
+    CryptoApproveAllowanceTransactionBody cryptoApproveAllowance = 48;
```

Comment:
It seems like this file is in lexicographic order, not ordinal order. We should move these to be sorted next to the other Crypto bodies. Would also be nice to add a note at the bottom about ordering and how to choose the next ordinal when adding new bodies.

Reply:
Moved these to the `Crypto*` group and added a comment.